.github/workflows/release.yaml

```diff
@@ -43,6 +43,9 @@ jobs:
 
     - id: action_vars
       run: echo ::set-output name=TAG::${GITHUB_REF#refs/tags/}
+    - uses: docker/setup-qemu-action@v1
+      with:
+        platforms: arm64
```

Comment:
doesn't this breaks `amd64` build then?

Reply:
Don't think so as the GH actions runners run on `amd64`. I could add `amd64` just to be safe? 
The whole GH actions build would need to be tested by you guys

Reply:
I'm not 100% sure if quemu is needed in the first place

Reply:
ahh, looks like this whole `with` block [is not needed](https://www.winters.nz/docker/cross-platform-github-actions/), could you please remove it?

Reply:
thanks, we'll see if it works or not :)